### PR TITLE
Remove features which are not supported by v2 components

### PR DESCRIPTION
**Pull request by notlee**
- Requires node 12 or later.
- Removes the ability to compare bundle sizes of any ref pushed to
Github. It is now only possible to compare the bundle sizes of
releases. This is because the [Origami Build Service](https://www.ft.com/__origami/service/build/) drops support
for bundling commit references for components which follow v2 of
the Origami Component Specification.
- Requires 3 arguments, the component name and two versions to compare.

closes: https://github.com/Financial-Times/origami-bundle-size-cli/issues/161

### CODE REVIEW

**Comment by notlee** [comment]
These changes mean it no longer supports the PR comments we are used to

File: MIGRATION.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Migration Guides
+
+## Migrating from v1 to v2
+
+- Requires node 12 or later.
+- Removes the ability to compare bundle sizes of any ref pushed to Github. It is now only possible to compare the bundle sizes of releases. This is because the [Origami Build Service](https://www.ft.com/__origami/service/build/) drops support for bundling commit references for components which follow v2 of the Origami Component Specification.
```
Comment (JakeChampion) [inline MIGRATION.md:6]: Good feature to remove, it does remove the ability to comment on pull-requests with the changes in js and css size but we can solve that another way.

One way to solve it is by publishing to npm every time a commit is push to a pull-request. The version in package.json could be the commit hash and we'd need to set the dist-tag to something other than 'latest' like we do for prereleases.

**Review by JakeChampion — APPROVED** [review]

File: MIGRATION.md (hunk 1 above)
Comment (notlee) [reply MIGRATION.md:6]: 👍 or I also liked @chee's idea, have obt figure out bundle sizes and commit results to the component

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v2.0.0 :tada:

**Comment by notlee** [comment]
☝️ fine, whatever, I didn't want a beta (I can live with this)

**Pull request merged**